rancher/v1.5/en/rancher-services/scheduler/index.md

```diff
@@ -0,0 +1,75 @@
+---
+title: Scheduling Services in Cattle Environments
+layout: rancher-default-v1.5
+version: v1.5
+lang: en
+---
+
+## Scheduling Services
+---
+
+Rancher's core scheduling logic is part of Rancher, which handles port conflicts and ability to schedule based on labels on hosts/containers. In addition to the core scheduling logic, Rancher supports additional scheduling policies by using the **Rancher Scheduler** catalog item.
+
+* [Ability to schedule against multiple IPs on a Host](#multiple-ips)
+* [Ability to scheduler based on resource constraints (i.e. CPU and memory)](#resource-constraints)
```

Comment:
Technically speaking, resource based scheduling contains not only CPU and memory but also container spreading across the hosts. So actually `Spreading containers across hosts in `active` state` is a subset of resource-constraints